src/components/global/BreadCrumbs.vue

```diff
@@ -0,0 +1,111 @@
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
+
+import BaseButton from '@/components/global/BaseButton.vue';
+import ArrowLeftIcon from '@/assets/icons/arrow-left-icon.svg?component';
+import ChevronRightIcon from '@/assets/icons/chevron-right-icon.svg?component';
+
+const router = useRouter();
+
+interface Crumb {
+  label: string;
+  route?: string;
+}
+
+interface Props {
+  crumbs?: Crumb[];
+}
+
+defineProps<Props>();
+
+function goBack() {
+  router.go(-1);
+}
+</script>
+
+<template>
+  <div class="bread-crumbs">
+    <BaseButton
+      colorScheme="dark-gray-text"
+      size="medium"
+      @click="goBack"
+      class="bread-crumbs__back-btn"
+    >
+      <ArrowLeftIcon />
+      <span class="bread-crumbs__back-btn__label">Back</span>
+    </BaseButton>
+    <div v-if="crumbs" class="bread-crumbs__items">
+      <template v-for="(crumb, index) in crumbs" :key="index">
+        <component
+          :is="crumb.route ? 'router-link' : 'span'"
+          :to="crumb.route !== '' ? { name: crumb.route } : ''"
```

Comment:
You could do it just like above, because empty strings are falsy 😄 
```suggestion
          :to="crumb.route ? { name: crumb.route } : ''"
```

Reply:
Oh, that's right... My brain must have turned off due to the late hour 😅 
Good eyes! 👀